textattack/commands/attack/attack_args_helpers.py

```diff
@@ -1,7 +1,6 @@
 import argparse
 import copy
 import importlib
-import json
```

Comment:
nice find! thanks for removing. 

can you remove `['argparse', 'pickle', 'sys', 'numpy', 'torch']`?

I don't know how all these got into this file, haha, but I don't think any of them are used.